packages/playwright/src/playwright.ts

```diff
@@ -20,7 +20,7 @@ export const clickAndInputCDPElement = async (page: Page, args: { id: string, va
 // Actions using Location
 export const hoverLocation = async (page: Page, args: { x: number, y: number }) => {
   const { element, tagName, isCustomElement } = await getElementAtLocation(page, args)
-  if (tagName === 'CANVAS' || isCustomElement) {
```

Comment:
We'll now return a `null` element rather than throwing if we cannot find one so that our fallback raw device input can kick in